api/vpc_peering.go

```diff
@@ -29,6 +30,11 @@ func (api *API) waitForPeeringStatus(instanceID int, peeringID string) (map[stri
 }
 
 func (api *API) ReadVpcInfo(instanceID int) (map[string]interface{}, error) {
+	// Initiale values, 5 attempts and 20 second sleep
+	return api.readVpcInfo(instanceID, 5, 20)
+}
+
+func (api *API) readVpcInfo(instanceID, attempts, sleep int) (map[string]interface{}, error) {
```

Comment:
Maybe help future code readers with a clear name?

```suggestion
func (api *API) readVpcInfoWithRetry(instanceID, attempts, sleep int) (map[string]interface{}, error) {
```